### DISCUSSION TITLE
creating a page <title>

### DISCUSSION THREAD

**Discussion by Livparran** [community/community#102074] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

when i go  into my settings there is supposed to be a automatic page generator under my settings for github pages but there is not. does anyone know where I can find that button? Thank you in advance.

**Comment by naveen-98** [comment]
Hello!

If you are unable to find the automatic page generator for GitHub Pages in your settings, there could be a few reasons for this. Here are some steps you can take to troubleshoot and find the solution:

1. **Check Repository Type:**
   - Ensure that your GitHub repository is a **public repository**. The GitHub Pages feature may not be available for private repositories.

2. **GitHub Repository Settings:**
   - Navigate to your GitHub repository on the web.
   - Click on the "Settings" tab, and then scroll down to the "GitHub Pages" section. The automatic page generator option should be visible here.

3. **GitHub Account Permissions:**
   - Make sure that your GitHub account has the necessary permissions to create and manage GitHub Pages. You may need to be the repository owner or have admin access.

4. **Repository Branch:**
   - Check the branch you are using for GitHub Pages. By default, GitHub Pages uses the `main` or `master` branch. Ensure that the branch you want to use for GitHub Pages is set correctly.

5. **Empty Repository:**
   - If your repository is completely empty, GitHub Pages may not be available. Consider adding some content or files to your repository.

6. **Browser Cache and Cookies:**
   - Clear your browser cache and cookies or try accessing GitHub from an incognito/private browsing window to rule out any browser-related issues.

If the issue persists, you might consider reaching out to GitHub support or community forums for further assistance.

**Comment by Livparran** [comment]
Hello I have checked all these things and I am still unable to automatically generate a page. 





On Saturday, January 27, 2024, 12:42 PM, Naveen Wijesinghe ***@***.***> wrote:




If you are unable to find the automatic page generator for GitHub Pages in your settings, there could be a few reasons for this. Here are some steps you can take to troubleshoot and find the solution:
   
   -    
Check Repository Type:
      
      - Ensure that your GitHub repository is a public repository. The GitHub Pages feature may not be available for private repositories.

   -    
GitHub Repository Settings:
      
      - Navigate to your GitHub repository on the web.
      - Click on the "Settings" tab, and then scroll down to the "GitHub Pages" section. The automatic page generator option should be visible here.

   -    
GitHub Account Permissions:
      
      - Make sure that your GitHub account has the necessary permissions to create and manage GitHub Pages. You may need to be the repository owner or have admin access.

   -    
Repository Branch:
      
      - Check the branch you are using for GitHub Pages. By default, GitHub Pages uses the main or master branch. Ensure that the branch you want to use for GitHub Pages is set correctly.

   -    
Empty Repository:
      
      - If your repository is completely empty, GitHub Pages may not be available. Consider adding some content or files to your repository.

   -    
Browser Cache and Cookies:
      
      - Clear your browser cache and cookies or try accessing GitHub from an incognito/private browsing window to rule out any browser-related issues.


If the issue persists, you might consider reaching out to GitHub support or community forums for further assistance.
